CMSgov/ato-blueprint
# ISPGBSS-716 update branding

https://jiraent.cms.gov/browse/ISPGBSS-716

### Changes
- Updated header and footer with new logo and updated links to match designs
- Moved navbar, home-user, and footer templates from the [ato-branding](https://github.com/CMSgov/rato-grq-branding) repo into this repo
- Updated browser tab with new icon and title
- Misc text changes from GovReady to Blueprint throughout the repo

### Testing
- Verify that the header and footer are consistent with designs in [Figma](https://www.figma.com/file/Q0BD09RJhn8TMdI8BwhiSS/Work-In-Progress?node-id=403%3A10072)
- Verify Browser tab displays new icon and text

_Note:_ The `Help` link in the header and the `Blueprint Support` link in the footer will both need to link to the new help pages that will be created in a future story. They are both currently linking to the existing support page.
